core/cloudflow-operator/src/test/scala/cloudflow/operator/TestDeploymentContext.scala

```diff
@@ -42,14 +42,14 @@ trait TestDeploymentContext {
       sparkRunnerSettings =
         SparkRunnerSettings(
           driverSettings = SparkPodSettings(
-            cores = Some("0.2"),
+            cores = Some(1),
```

Comment:
Quite a jump!

Reply:
Yeah. The new spark operator CRD requires this number to be an integer that’s >= 1.

Reply:
Ah, ok. Do you have a link to background info on this? why did they do this and what's the potential impact on our minimum resource usage?

Reply:
I can't seem to find the rationale behind this change. FYI, this is where the restriction is imposed: https://github.com/GoogleCloudPlatform/spark-on-k8s-operator/blob/master/manifest/crds/sparkoperator.k8s.io_sparkapplications.yaml#L345